### PR TITLE
Adding Integration Tests to Manager

### PR DESCRIPTION
**Pull request by matheuscmelo**
This PR is able to add integration tests to manager's CI.

### CODE REVIEW

**Review by liliarsampaio — APPROVED** [review]
thanks, lgtm.

**Review by clenimar — APPROVED** [review]
why is the CI failing here? looking forward to seeing it up and running. thank you.

**Review by javanlacerda — APPROVED** [review]
I think that the failure is because the cluster is full. However is good to check it!

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,29 @@
 pipeline {
   agent any
   stages {
-    stage('') {
+    stage('Unit') {
+      agent any
       steps {
         sh 'tox'
```
Comment (rafaelvfalc) [inline Jenkinsfile]: Tox should not recreate the virtual environment in every build?

Comment (matheuscmelo) [reply Jenkinsfile]: Since Jenkins create a new workspace for each build, it doesn't need to recreate the env because there's no env in the first place

**Review by rafaelvfalc — APPROVED** [review]
Looks good, just pointed a minor question about 'tox' command. Thank you for the contribution!

**Pull request merged**